package.json

```diff
@@ -118,7 +118,7 @@
     "@types/ws": "^4.0.1",
     "@types/yargs": "^15.0.11",
     "@types/yargs-parser": "^15.0.0",
-    "@typescript-eslint/parser": "^4.7.0",
+    "@typescript-eslint/parser": "4.16.1",
```

Comment:
nothing substantial changed in this, there's been [so many releases](https://github.com/typescript-eslint/typescript-eslint/releases) because it shares a monorepo with the actual typescript-eslint plugin. If we don't update this, though, we get a giant 
```
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.2.0

YOUR TYPESCRIPT VERSION: 4.2.2
```
warning on every run of `yarn lint` :)